website/pages/en/index.js

```diff
@@ -196,7 +196,7 @@ class AwkwardScrollingImageWithText extends Component {
                 </MarkdownBlock>
               </div>
             </div>
-            <img src="https://media.giphy.com/media/13WZniThXy0hSE/giphy.gif" />
+            <img src="https://media.giphy.com/media/13WZniThXy0hSE/giphy.gif" alt="hot-loader" />
```

Comment:
`alt` attributes should be natural language descriptions in proper casing (they may be used my screen readers). So it would be more appropriate to call it "Hot reloading demo".

Reply:
Sure I will update  it soon..